website/views/options/social/tavern.jade

```diff
@@ -71,12 +71,9 @@
             |<a href='/#/options/groups/hall'>
             =env.t('visitHeroes')
             |</a> <br />
-            |<a href='http://habitica.wikia.com/wiki/Contributor_Rewards' target='_blank'>
-            =env.t('conLearn')
-            |</a> <br />
-            |<a href='http://habitica.wikia.com/wiki/Contributing_to_Habitica' target='_blank'>
-            =env.t('conLearnHow')
-            |</a>
+            !=env.t('conLearn')
+            a(href=env.t('conLearnLink'), target='_blank')
+              =env.t('conLearnHow')
```

Comment:
The  `<br />` tag is still needed:

![image](https://cloud.githubusercontent.com/assets/1495809/17537358/5670bd3c-5edf-11e6-94c1-bad3acf67d5e.png)